requirements.txt

```diff
@@ -0,0 +1,29 @@
+appdirs==1.4.3
```

Comment:
pip freeze > requirements.txt.
Allows for more reproducible environments.

Yes, auto generated.